spec/services/gov_delivery/client_spec.rb

```diff
@@ -1,8 +1,5 @@
 require "rails_helper" # These tests use #strip_heredoc which is in ActiveSupport
 
-require "app/services/gov_delivery/client"
-require "app/services/gov_delivery/request_builder"
```

Comment:
In that case could you move them into a separate commit to make it clear that it's unrelated to this change?